src/main/java/com/basho/riak/client/core/RiakNode.java

```diff
@@ -136,7 +136,7 @@ public void operationComplete(ChannelFuture future) throws Exception
                 // the next time it's pulled from the pool.
                 // We record the disco for the health check.
                 recentlyClosed.add(new ChannelWithIdleTime(future.channel()));
-                logger.error("inAvailable channel closed; id:{} {}:{}",
+                logger.warn("Available channel closed; id:{} {}:{}",
```

Comment:
@alexmoore Not sure whether it should be a warning, since it is an ordinal situation